administrator/components/com_csp/src/Helper/ReporterHelper.php

```diff
@@ -76,4 +76,62 @@ public static function getCspTrashStatus()
 
 		return boolval($result);
 	}
+
+	/**
+	 * Check whether there are unsafe-inline rules published
+	 *
+	 * @return  boolean  Whether there are unsafe-inline rules published
+	 *
+	 * @since   4.0.0
+	 */
+	public static function getCspUnsafeInlineStatus()
+	{
+		$db = Factory::getDbo();
+		$query = $db->getQuery(true)
+			->select('COUNT(*)')
+			->from($db->quoteName('#__csp'))
+			->where($db->quoteName('blocked_uri'). ' = ' . $db->quote("'unsafe-inline'"))
+			->where($db->quoteName('published') . ' = ' . $db->quote('1'));
+		$db->setQuery($query);
+
+		try
+		{
+			$result = (int) $db->loadResult();
+		}
+		catch (\RuntimeException $e)
+		{
+			Factory::getApplication()->enqueueMessage($e->getMessage(), 'error');
+		}
+
+		return boolval($result);
+	}
+
+	/**
+	 * Check whether there are unsafe-eval rules published
+	 *
+	 * @return  boolean  Whether there are unsafe-eval rules published
+	 *
+	 * @since   4.0.0
+	 */
+	public static function getCspUnsafeEvalStatus()
+	{
+		$db = Factory::getDbo();
+		$query = $db->getQuery(true)
+			->select('COUNT(*)')
+			->from($db->quoteName('#__csp'))
+			->where($db->quoteName('blocked_uri'). ' = ' . $db->quote("'unsafe-eval'"))
```

Comment:
```suggestion
			->where($db->quoteName('blocked_uri') . ' = ' . $db->quote("'unsafe-eval'"))
```
Drone PHPCS: "Concat operator must be surrounded by a single space."